### ISSUE TITLE
Hierarcy slicher accessibility

### ISSUE THREAD

**Issue by elijoho** [liprec/azurebi-docs#46]
The regular slicer in Power BI have support for klicking CTRL + Right arrow to "enter" the filter. Therby you can control the filter using only keyboard. 
This does not work for hierarcy slicer and it is therefore not possible to use this awesome visual when creating reports that need to meet accessibility standards. 

---
#### Document details

⚠ Do not edit this section. It is required for docs.jppp.org ➟ GitHub issue linking.

* document_id=487cd175-be13-40df-a80e-aa1c7f531eb1
* document_url=https://azurebi-docs.jppp.org/powerbi-visuals/hierarchy-slicer.html
